movement/movement.c

```diff
@@ -149,7 +153,16 @@ static void _movement_handle_scheduled_tasks(void) {
 
 void movement_request_tick_frequency(uint8_t freq) {
     if (freq == 128) return; // Movement uses the 128 Hz tick internally
-    RTC->MODE2.INTENCLR.reg = 0xFE; // disable all callbacks except the 128 Hz one
+
+    // disable all callbacks except the 128 Hz one
+#if __EMSCRIPTEN__
+    for (int i = 1; i < 128; i = i << 1) {
+        watch_rtc_disable_periodic_callback(i);
+    }
+#else
+    RTC->MODE2.INTENCLR.reg = 0xFE;
+#endif
```

Comment:
Watch faces should be discouraged from that, for the exact reason you point out here: it will mean hardware-specific overrides. Having said that I think movement.c is an exception since it's not a watch face, it's the plumbing that runs the whole Movement system. 

This is a kind of a specific hack that I should probably abstract out to a function on the RTC class, it's just challenging because the presence of eight prescaler interrupts is a pretty specific feature of this hardware.